spec.html

```diff
@@ -6796,10 +6811,10 @@ <h1>GetIdentifierReference ( _env_, _name_, _strict_ )</h1>
         <p>The abstract operation GetIdentifierReference takes arguments _env_ (an Environment Record or *null*), _name_ (a String), and _strict_ (a Boolean). It performs the following steps when called:</p>
         <emu-alg>
           1. If _env_ is the value *null*, then
-            1. Return a value of type Reference whose base value component is *undefined*, whose referenced name component is _name_, and whose strict reference flag is _strict_.
+            1. Return Reference Record { [[Base]]: ~empty~, [[ReferencedName]]: _name_, [[Strict]]: _strict_, [[ThisValue]]: ~empty~ }.
```

Comment:
Here and elsewhere, "Return Reference Record { }" reads odd. I'd prefer "Return the Reference Record { }", since we're referring Reference Records with Record in the type name. (As contrasted with something like Completion Records, which we just call Completions.)

Reply:
Fixed in 2e92d58.